### PR TITLE
Add DockerImage Kind in BuildOutput

### PR DESCRIPTION
**Pull request by eye0fra**
## Description
Address the issue defined https://github.com/fabric8io/fabric8-maven-plugin/issues/1054

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [X] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [X] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [x] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [X] I have implemented unit tests to cover my changes
 - [x] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [x] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [X] I tested my code in OpenShift

### CODE REVIEW

**Review by manusa — APPROVED** [review]
LGTM, thx!

File: jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/openshift/OpenshiftBuildServiceTest.java [hunk 1]
```diff
@@ -509,6 +563,62 @@ public void testBuildConfigResourceConfig() throws Exception {
...
+            WebServerEventCollector collector = createMockServer(config, true, 50, false, false);
+
+            DefaultOpenShiftClient client = (DefaultOpenShiftClient) mockServer.getOpenshiftClient();
+            LOG.info("Current write timeout is : {}", client.getHttpClient().writeTimeoutMillis());
+            LOG.info("Current read timeout is : {}", client.getHttpClient().readTimeoutMillis());
+            LOG.info("Retry on failure : {}", client.getHttpClient().retryOnConnectionFailure());
+            OpenshiftBuildService service = new OpenshiftBuildService(client, logger, jKubeServiceHub);
+            service.build(image);
+
+            // we should Foadd a better way to assert that a certain call has been made
```
Comment (rohanKanojia) [inline jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/openshift/OpenshiftBuildServiceTest.java:612]:
```suggestion
            // we should add a better way to assert that a certain call has been made
```

File: jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/openshift/OpenshiftBuildServiceTest.java [hunk 2]
```diff
@@ -509,6 +563,62 @@ public void testBuildConfigResourceConfig() throws Exception {
...
```
Comment (rohanKanojia) [inline jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/openshift/OpenshiftBuildServiceTest.java:585]:
```suggestion
            // we should add a better way to assert that a certain call has been made
```

**Review by rohanKanojia — APPROVED** [review]

**Pull request merged**